pkg/operator/ceph/nfs/config.go

```diff
@@ -47,18 +48,22 @@ func getNFSNodeID(n *cephv1.CephNFS, name string) string {
 	return fmt.Sprintf("%s.%s", n.Name, name)
 }
 
-func getGaneshaConfigObject(clusterName string) string {
-	return fmt.Sprintf("conf-nfs.%s", clusterName)
+func getGaneshaConfigObject(n *cephv1.CephNFS, version cephver.CephVersion, name string) string {
```

Comment:
What's the difference between `n.Name` and `name`?

Reply:
`n.Name` is the cluster name and `name` is the node id. I did not use nodeid instead of name to avoid confusion. Since we define nodeid again here:https://github.com/rook/rook/blob/afcfe40b5e0efef01ca66a6569e36e482c990951/pkg/operator/ceph/nfs/config.go#L46-L49